db/dcrsqlite/apisource.go

```diff
@@ -40,15 +40,17 @@ type wiredDB struct {
 	params   *chaincfg.Params
 	sDB      *stakedb.StakeDatabase
 	waitChan chan chainhash.Hash
+	liteMode bool
```

Comment:
wiredDB should NOT be aware of the modes of explorer.  The notion of lite/full mode stays in explorer/api.  Instead of this struct field being called `liteMode`, name it according to its purpose, e.g. `updateSyncStatus`.

Reply:
This `liteMode` is needed to determine which of the `sync.go` file either from `dcrpg` or `dcrsqlite` package should be updating the progress that is running on the sync status page.

[Here](https://github.com/decred/dcrdata/pull/710/files#diff-da97fde754870fa5c4b02913ac225e8aR249) is one instance where its important.  On Lite Mode `func (db *wiredDB) resyncDB(` method in `sync.go` from `dcrsqlite` package sends the updates  while  on full mode `func (db *ChainDB) SyncChainDB(` in `sync.go` from `dcrpg`package sends the updates.

It would be so unfortunate and confusing to have been methods sending progress updates especially when Full Mode is running since both functions must run.

Reply:
I will change it to that name.

Reply:
The types implement different interfaces. In package api, `wiredDB` implements the `DataSourceLite` interface, while `ChainDB` implements `DataSourceAux`.  So each type has a clear role w.r.t. modes.  I think it's fair to put a comment in the docs for `InitWiredDB` explaining that a case when `wiredDB` should be responsible for updating sync status is if the auxiliary / full-mode DB is not handling the task.